preproc.c

```diff
@@ -215,6 +215,7 @@ static int include_file(struct cpp* cpp, struct tokenizer *t, FILE* out) {
 	struct token tok;
 	tokenizer_set_flags(t, 0); // disable string tokenization
 
+    char * dirname = NULL;
```

Comment:
Sorry, just noticed I missed a tab here (my editor is configured for spaces by default):

```suggestion
	char * dirname = NULL;
```